includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -61,16 +61,18 @@ class WC_Stripe_Payment_Request {
 	/**
 	 * Initialize class actions.
 	 *
-	 * @since 3.0.0
+	 * @since   3.0.0
 	 * @version 4.0.0
 	 */
 	public function __construct() {
-		self::$_this            = $this;
-		$this->stripe_settings  = get_option( 'woocommerce_stripe_settings', array() );
-		$this->testmode         = ( ! empty( $this->stripe_settings['testmode'] ) && 'yes' === $this->stripe_settings['testmode'] ) ? true : false;
-		$this->publishable_key  = ! empty( $this->stripe_settings['publishable_key'] ) ? $this->stripe_settings['publishable_key'] : '';
-		$this->secret_key       = ! empty( $this->stripe_settings['secret_key'] ) ? $this->stripe_settings['secret_key'] : '';
-		$this->total_label      = ! empty( $this->stripe_settings['statement_descriptor'] ) ? WC_Stripe_Helper::clean_statement_descriptor( $this->stripe_settings['statement_descriptor'] ) : '';
+		self::$_this                        = $this;
+		$this->stripe_settings              = get_option( 'woocommerce_stripe_settings', array() );
+		$this->testmode                     = ( ! empty( $this->stripe_settings['testmode'] ) && 'yes' === $this->stripe_settings['testmode'] ) ? true : false;
+		$this->publishable_key              = ! empty( $this->stripe_settings['publishable_key'] ) ? $this->stripe_settings['publishable_key'] : '';
+		$this->secret_key                   = ! empty( $this->stripe_settings['secret_key'] ) ? $this->stripe_settings['secret_key'] : '';
+		$this->total_label                  = ! empty( $this->stripe_settings['statement_descriptor'] ) ? WC_Stripe_Helper::clean_statement_descriptor( $this->stripe_settings['statement_descriptor'] ) : '';
+		// Defaults to 'required' to match how core initializes this option.
+		$this->require_phone_with_apple_pay = 'required' === get_option( 'woocommerce_checkout_phone_field', 'required' );
```

Comment:
If it's no longer reading from `$this->stripe_settings`, I'm not sure it belongs here – and since it's only used in one place, there might not be much reason to set it on the instance – thoughts?

Reply:
Yeah, you're right. I'll move this to where it's loaded into the JS. Thanks! ❤️

Reply:
Fixed in 16900225601defc539074658623a25cf6a3a7aed 🎉